rmf_demo_plugins/rmf_plugins_common/src/ingestor_common.cpp

```diff
@@ -0,0 +1,92 @@
+#include <rmf_plugins_common/ingestor_common.hpp>
+
+namespace rmf_ingestor_common {
+
+TeleportIngestorCommon::TeleportIngestorCommon()
+{
+}
```

Comment:
It seems there is quite a few cases of constructors / destructors being empty. I would suggest either changing them to
```
public:
  ObjectName() = default;
```
if you want to be explicit about them being default constructors or omit them altogether.